schemas/Object.yaml

```diff
@@ -309,10 +309,6 @@ funcs:
         functor: Column
         args: slot_ShapeRound_xy
         dataset: meas
-    iPSF_flag:
```

Comment:
Why are you removing this?

Reply:
This is defined earlier here: https://github.com/lsst/pipe_tasks/blob/fb4685b71f45d7d9c43d08f860a877ec8270831b/schemas/Object.yaml#L292-L295

I'm removing the duplicate entry in this commit.